### ISSUE TITLE
Be able to update a wishlist when sharing it

### ISSUE THREAD

**Issue by boboldehampsink** [verbb/wishlist#133]
### What are you trying to do?

I would like to see that passing, for example, `fields[custom]` to `share-by-email` would not only include it in the email template, but also save it to the wishlist (provided this field is in the fieldlayout).

### What's your proposed solution?

Add the saving of the list element to the share by email controller action

**Comment by engram-design** [comment]
I'll see what I can do here, but I would think that sharing a list is a specific action to share, and not to update anything about the list or items. But I can see how it might make sense to do so.

We do currently [support](https://github.com/verbb/wishlist/blob/2bb587885b1652b57eb0e6457a990033978197de/src/controllers/ListsController.php#L457) having `fields` available in your email template - just in case that's not working.

**Comment by boboldehampsink** [comment]
@engram-design I went ahead and created a PR for this (#134) - It doesn't really change the functionality of sharing, only when the list is yours it gets updated.

I have a use-case for this in which a client can request a quote of their wishlist - and it works by explicitly sharing their wishlist with the shop owner. But for that they also have to add contact data, which I've added fields for in the wishlist field layout. With this update, I can share the list with the shop owner and also save contact data to it at the same time.
